tests/conftest.py

```diff
@@ -557,16 +557,16 @@ def minicircle_grain_threshold_abs(minicircle_grains: np.array) -> Grains:
 
 
 @pytest.fixture
-def minicircle_grain_mask(minicircle_grain_threshold_otsu: Grains) -> Grains:
+def minicircle_grain_mask(minicircle_grain_threshold_abs: Grains) -> Grains:
```

Comment:
Why the switch from `otsu > abs` for threhsolding? Is it so that you can get the necessary small grains to test removal of small objects?

Reply:
Yeah 😄